include/deal.II/numerics/fe_field_function.templates.h

```diff
@@ -260,7 +260,7 @@ namespace Functions
 
     unsigned int ncells = compute_point_locations(points, cells, qpoints, maps);
     hp::MappingCollection<dim> mapping_collection (mapping);
-    hp::FECollection<dim> fe_collection (dh->get_fe ());
+    hp::FECollection<dim> fe_collection (dh->get_fe_collection());
```

Comment:
Here (and in a number of places further down) you do not need to explicitly construct an `FECollection` any more, you can simply use the one by `dh->get_fe()`.